nemo/core/classes/exportable.py

```diff
@@ -60,6 +60,7 @@ def export(
         dynamic_axes=None,
         check_tolerance=0.01,
         export_modules_as_functions=False,
+        keep_initializers_as_inputs=False,
```

Comment:
Please default to None, and we can merge

Reply:
Done.

Reply:
Thanks for the PR!